frontend/components.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "https://ui.shadcn.com/schema.json",
+  "style": "default",
+  "rsc": false,
+  "tsx": true,
+  "tailwind": {
+    "config": "tailwind.config.js",
+    "css": "src/index.css",
+    "baseColor": "slate",
+    "cssVariables": true,
+    "prefix": ""
+  },
+  "aliases": {
+    "components": "@/components",
+    "utils": "@/lib/utils"
```

Comment:
```suggestion
    "components": "~/components",
    "utils": "~/lib/utils"
```

Reply:
I prefer to use the tilde (`~`) character for relative imports instead of the at (`@`) symbol because we might confuse them with scoped libraries like `@tanstack/react-query` for ex, with `~` it more obvious that the imports are from the the local relative path instead of coming from some lib.